### PR TITLE
Update bindgen version

### PR DESCRIPTION
**Pull request by westy92**
This is causing conflicts with other libraries.

### CODE REVIEW

**Comment by filmil** [comment]
Sorry for the delay. I was AFK for a few weeks.

**Comment by filmil** [comment]
Yes, but as you can see it's also finnicky about rust version, so...

**Comment by westy92** [comment]
If support for older rust versions is required, I can instead just upgrade bindgen to a newer version.

File: rust_icu_sys/src/lib.rs [hunk 1]
```diff
@@ -13,18 +13,13 @@
 // limitations under the License.
 #![doc(test(ignore))]
 
-// Notes:
-// * deref_nullptr: since rustc 1.53, bindgen causes UB warnings -- see
-// https://github.com/rust-lang/rust-bindgen/issues/1651 remove this once bindgen has fixed the
-// issue (currently at version 1.59.1)
```
Comment (westy92) [inline rust_icu_sys/src/lib.rs:19]: This appears to have been fixed in bindgen 0.60.

File: build/Dockerfile.testenv [hunk 2]
```diff
@@ -21,7 +21,7 @@ RUN mkdir -p $RUST_ICU_SOURCE_DIR && \
 COPY entrypoint.sh /entrypoint.sh 
 RUN chmod a+rwx /entrypoint.sh
 
-RUN cargo install --force --version 0.59.2 bindgen
```
Comment (filmil) [inline build/Dockerfile.testenv:24]: This should be pegged to a specific bindgen version. Here and elsewhere.

**Comment by westy92** [comment]
Woot! Happy to see some green checkmarks. 🎉

**Comment by westy92** [comment]
Would you be able to release this in a 4.0.1?

**Comment by filmil** [comment]
Sure. Can you please file a bug for me so (1) I don't forget, and (2) you get a progress tracker for that fix.

**Comment by westy92** [comment]
Done here: https://github.com/google/rust_icu/issues/273

**Comment by filmil** [comment]
Fixed via https://github.com/google/rust_icu/pull/277

**Pull request merged**